tests/foreman/ui/test_host.py

```diff
@@ -372,6 +359,37 @@ def module_libvirt_hostgroup(
     ).create()
 
 
+@pytest.fixture(scope='module')
+def manifest_org(module_org):
+    """Upload manifest to organization."""
+    with manifests.clone() as manifest:
+        upload_manifest(module_org.id, manifest.content)
+    return module_org
+
+
+@pytest.fixture(scope='module')
+def module_activation_key(manifest_org):
+    """Create activation key using default CV and library environment."""
+    activation_key = entities.ActivationKey(
+        auto_attach=True,
+        content_view=manifest_org.default_content_view.id,
+        environment=manifest_org.library.id,
+        name=gen_alpha(),
```

Comment:
I think random name is generated by default, see examples in `pytest_fixtures/api_fixtures.py`